chain/network/src/peer/peer_actor.rs

```diff
@@ -418,7 +425,7 @@ impl PeerActor {
             if let Some(peer_id) = self.other_peer_id() { peer_id.clone() } else { return };
 
         // Wrap peer message into what client expects.
-        let network_client_msg = match msg {
+        let network_client_msg = match msg.clone() {
```

Comment:
we're doing a bunch of "clone" calls -- let's make sure that we don't duplicate something expensive (like large incoming network message)